CVRParamLib/ModPluginLoader.cs

```diff
@@ -214,4 +214,19 @@ class AvatarDetails_tHook
         [HarmonyPrefix]
         static void Recycle(AvatarDetails_t __instance) => AvatarHandler.CacheAvatar(__instance);
     }
+    
+    [HarmonyPatch(typeof(PlayerSetup), "changeAnimatorParam")]
+    class ChangeAnimatorParam_Hook
+    {
+        [HarmonyPostfix]
+        static void changeAnimatorParam(string name, float value) {

```

Comment:
This is the same method that is used to change animator parameters over OSC, that of which will cause looping issues. On top of this, some mods that drive Animator Parameters will not use this method, and will use the CVRAnimatorManager instead.

Reply:
I mean if someone changes a parameter via OSC it still should trigger a parameter change OSC msg.

User sends param change OSC msg `->` mod changes the param using changeAnimatorParam method `->` mod send OSC msg saying the param was changed .

Or am I missing something?

As for the other mod compatibility you can't cover all things mods are going to do, because each develops their own way. But I hooked deeper in the change parameter flow. This should cover all mods, unless you seen a very specific one that changes even deeper. I think deeper than this would miss on the parameter sync logic.

Reply:
The deeper hook seems better for other mods in-case of compatibility. The flow was fine from the start, I was just worried about other mods not using `changeAnimatorParam`